app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,19 @@
+<div class="<%= dom_id(sender)%>_<%= dom_id(recipient) %>_follow_unfollow">
```

Comment:
Great job making the change by adding a div tag to capture the follow request dom_id !